makefile

```diff
@@ -378,6 +378,18 @@ phony += clean
 gqlgen:
 	go run github.com/99designs/gqlgen generate
 
+swaggo-install:
+	go install github.com/swaggo/swag/cmd/swag@latest
+
+swaggo-format:
+	swag fmt -g service/service.go
+
+swaggo-build:
+	swag init -g service/service.go
+
+swaggo-render:
+	npx @redocly/cli build-docs docs/swagger.json
```

Comment:
I don't know if people really all have it installed/have it installed to a specific version if you need that. This should be added to the list of things to install in the new hire guide + tell the other engineers.

Reply:
Can we add it to [the list of stuff that new hires have to install](https://docs.google.com/document/d/1LJHPjLb_bUZxv-l_9Y61kitzAg7u-13I5DjOhZ-deBA/edit#heading=h.ckpfbbfdjtpm)?